.env.example

```
@@ -43,6 +43,7 @@ GOOGLE_MAPS_GEOCODING_API_KEY=
 
 # Redis instance URL
 REDIS_URL=redis://localhost:6379/0
+REDIS_CACHE_URL=redis://localhost:6379/0
```

```suggestion
REDIS_CACHE_URL=redis://localhost:6379/1
```

Suggest a different database

---

Hum, redis://localhost:6379/1 is already in use in config/cable.yml. Shouldn't we maybe change it?